client/src/views/Panel.tsx

```diff
@@ -1,11 +1,37 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
 import FuelLevelCard from '@/components/FuelLevel';
 import { Table } from '@/components/Table';
 import { Card } from '@/components/card';
 import Station1chart from '@/components/charts/Station1chart';
+import { getAllPredictions } from '@/dao/prediction';
+import { usePromise } from '@/hooks';
 
 import './Panel.css';
 
 const Panel = () => {
+  const { id } = useParams<{ id: string }>();
+
+  console.log(id);
+
+  const [predictions, setPredictions] = useState<unknown[]>([]);
```

Comment:
typ unknown, to nie jest dobra praktyka. Sprawdź co przyjdzie ci z backendu i utwórz stosowny typ